spec/services/preserved_object_handler_update_version_spec.rb

```diff
@@ -35,19 +35,24 @@
         let(:updated_pc_db_msg) { "#{exp_msg_prefix} PreservedCopy db object updated" }
         let(:updated_po_db_msg) { "#{exp_msg_prefix} PreservedObject db object updated" }
 
-        it "updates entries with incoming version" do
+        it "updates PreservedCopy with incoming version" do
           expect(pc.version).to eq 2
-          expect(po.current_version).to eq 2
           po_handler.update_version
+          expect(pc.reload.version).to be > 2
           expect(pc.reload.version).to eq incoming_version
+        end
+        it "updates PreservedObject with incoming version" do
+          expect(po.current_version).to eq 2
+          po_handler.update_version
+          expect(po.reload.current_version).to be > 2
```

Comment:
ditto above comment about this feeling unnecessary in light of the following line and me not feeling strongly about changing it